sdk/core/azure-core/azure/core/pipeline/_tools_async.py

```diff
@@ -31,3 +32,36 @@ async def await_result(func, *args, **kwargs):
         # type ignore on await: https://github.com/python/mypy/issues/7587
         return await result  # type: ignore
     return result
+
+def to_rest_response(pipeline_transport_response):
+    response_type = None
+    try:
+        from .transport import AioHttpTransportResponse
+        from ..rest._aiohttp import RestAioHttpTransportResponse
```

Comment:
Do we support (or plan to support) custom transport?

Reply:
so what I'm doing here is "if . However, I can't do a specific corresponding rest transport response for a custom transport since idk what that transport is, so I fall back to just creating the default `AsyncHttpResponse` out of it

Our goal in a later PR (hopefully in the next release cycle) is to fix the need to transform rest requests and responses, and somehow have our pipelines support both the old and new requests and responses. So this code is also temporary.